annoyed-alligators/manage.py

```diff
@@ -8,6 +8,10 @@ def main():
     os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'socl_media.settings')
     try:
         from django.core.management import execute_from_command_line
+
+        from django.core.management.commands.runserver import Command as runserver
+        runserver.default_port = "8080"
```

Comment:
I'd suggest you to maybe change this back to 8000 while submitting a PR as it will cause merge conflicts. You can change the port when running the server locally by `python manage.py 8080`. So no need to change it in the file.